package_control/downloaders/wget_downloader.py

```diff
@@ -235,8 +234,8 @@ def parse_output(self, clean_run):
             HTTP header names.
         """
 
-        with open_compat(self.tmp_file, 'r') as f:
-            output = read_compat(f).splitlines()
+        with open(self.tmp_file, 'r', encoding='utf-8', errors='replace') as fobj:
```

Comment:
Similar to curl, I looked into wget and couldn't find anything about the encoding of the log file. Since we are grabbing HTTP headers, we should probably decode as ISO-8859-1.

Reply:
Removed `errors='replace'` here as well. Hope that's ok?

Reply:
Yeah, honestly these are probably very rarely used these days now that ST 3.0+ has the SSL module built in on Linux